frontend/src/components/SavedQuery/index.tsx

```diff
@@ -46,13 +58,43 @@ const SavedQuery = ({
   const [isFetching, setIsFetching] = useState(false);
   const [executionResponse, setExecutionResponse] =
     useState<ExecutionResponse | null>(null);
+  const [queryType, setQueryType] = useState<String | null>(null);
```

Comment:
```suggestion
  const [queryType, setQueryType] = useState<string | null>(null);
```

Always use `string` in typescript.